lib/steep/project/dsl.rb

```diff
@@ -74,6 +74,10 @@ def check(*args)
           sources.concat(args)
         end
 
+        def ext(ext)
+          @ext = ext
+        end
```

Comment:
Is there any better name for this? I have some other ideas: `file_ext`, `source_ext` and `extensions` (plural).

Reply:
`ext` and `extensions` sound like “plugins”.
Here’s another: `check_ext`

Honestly, it should be called “`templating`”.
ERB templating syntax is not Ruby syntax, and just changing the extension won’t teach Steep how to recognize them, not to mention this design currently blocks a target from continuing checking `*.rb`.
If Steep can automatically switch to ERB mode for `*.erb`s, it can simply change the default file extension to `.{rb,erb}`.

Reply:
> If Steep can automatically switch to ERB mode for *.erbs, it can simply change the default file extension to .{rb,erb}.

Some template languages, such as HAML, can be converted to Ruby in real-world apps. Therefore it might be better to provide to enhance Steep via plug-ins. From that perspective, it would be better to specify the file extensions via Steepfile.

> P.S. If it’s just for pretending say .ruby files to be synonymous with .rb, is check 'app/views/**/*.{rb,ruby}' sufficient?

Unfortunately, it does not work perfectly.

Please see the diff of `lib/steep/server/master.rb`. It has a hard-coded `.rb` string to set up `didChangeWatchedFiles` event.